<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>448</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>티머니 둥근바람 Regular</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>프로젝트 추가</string>
  </property>
  <widget class="QListWidget" name="algorithm_listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>291</width>
     <height>341</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>티머니 둥근바람 Regular</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="algorithm_comboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>291</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="accept_pushButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>확인</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reject_pushButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>취소</string>
   </property>
  </widget>
  <widget class="QPushButton" name="del_alg_pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>선택 알고리즘 제거</string>
   </property>
  </widget>
  <widget class="QPushButton" name="detail_pushButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>40</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>세부 정보 설정</string>
   </property>
  </widget>
  <widget class="QComboBox" name="radar_comboBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>90</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>31</width>
     <height>371</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>70</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose Radar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
